Build.ps1

```diff
@@ -100,6 +100,14 @@ Properties {
     # will no longer be prompted for the API key.
     $NuGetApiKey = $null
     $EncryptedApiKeyPath = "$env:LOCALAPPDATA\vscode-powershell\NuGetApiKey.clixml"
+
+    # If you do not specify the certificate thumbprint when specifying a build that
+    # includes script signing the build will use the first code signing certificate
+    # it finds in the users personal certificate store, unless a path to a PFX file
+    # is specified using the CertPfxPath parameter. The build will import the 
+    # certificate (if required), then store the thumbprint encrypted in a file, so
+    # that on subsequent signing the build will use the same certificate.
+    $CertThumbprintPath  = "$env:LOCALAPPDATA\WindowsPowerShell\CertificateThumbprint.clixml"
 }
```

Comment:
Yeah, I am just storing the data using the built in function that turns it into a securestring, I just mentioned it was encrypted there in the comment just to be clear in case someone goes to look at the file and wonders what's up. I'm definitely going to pursue the change to subject name with latest valid date, I never thought about it that way round when I was doing it. Listing the certs and letting the user pick is a good one too, I was maybe a bit too focused on the unattended experience, but i'll keep that in mind.

Making the key value store a bit more general for data is definitely something that would be good. I thought about doing that first, but I already had most of this signing stuff around and in use, but it highlights the need for it at least.